src/platform/macos.rs

```diff
@@ -14,7 +14,7 @@ lazy_static! {
 	};
 }
 
-
+#[cfg(target_arch = "x86_64")]
 extern "stdcall" {
```

Comment:
I wonder why is this `stdcall`, I thought this is for windows. And macOS uses System V, no? Does extern "C" work?

Reply:
Good question, according to https://doc.rust-lang.org/stable/reference/items/external-blocks.html#abi  `System V` is default but it has worked in the past AFAIK.

//cc @debris

Reply:
Hmm, maybe it worked because this function was not called at all?